pkgs/applications/misc/megasync/default.nix

```diff
@@ -24,21 +25,30 @@
 , unzip
 , wget
 }:
-
 mkDerivation rec {
   pname = "megasync";
-  version = "4.3.5.0";
+  version = "4.4.0.0";
 
   src = fetchFromGitHub {
     owner = "meganz";
     repo = "MEGAsync";
     rev = "v${version}_Linux";
-    sha256 = "0rr1jjy0n5bj1lh6xi3nbbcikvq69j3r9qnajp4mhywr5izpccvs";
+    sha256 = "1xggca7283943070mmpsfhh7c9avy809h0kgmf7497f4ca5zkg2y";
     fetchSubmodules = true;
   };
 
   nativeBuildInputs =
-    [ autoconf automake doxygen lsb-release pkg-config qttools swig unzip ];
+    [
+      autoconf
+      automake
+      doxygen
+      libtool
+      lsb-release
+      pkg-config
+      qttools
+      swig
+      unzip
+    ];
```

Comment:
```suggestion
    autoconf
    automake
    doxygen
    libtool
    lsb-release
    pkg-config
    qttools
    swig
    unzip
  ];
```

Reply:
my bad for relying solely on nix-community/nixpkgs-fmt
addressed

Reply:
That shouldn't introduce this change by itself.

Reply:
Correct, I just the left bracket sitting on its original place one line below the assignment line.